src/config/dark-sites.config

```diff
@@ -340,6 +341,7 @@ ibookpile.net
 identinator.com
 ieuan.tech
 ieuan.xyz
+ign.com
```

Comment:
Not that dark either
![image](https://user-images.githubusercontent.com/25481501/109141888-2402fb00-7756-11eb-9ec5-6ccc11543f0d.png)

Reply:
I guess IGN International hasn't rolled out dark mode based on theme yet
<img width="1347" alt="image" src="https://user-images.githubusercontent.com/73370925/109157404-f7dc7f80-773f-11eb-8cb6-8c3fd1090b43.png">